pkg/scheduler/framework/plugins/defaultpreemption/default_preemption_test.go

```diff
@@ -337,6 +337,9 @@ func TestPostFilter(t *testing.T) {
 			}
 			// As we use a bare clientset above, it's needed to add a reactor here
 			// to not fail Victims deletion logic.
+			cs.PrependReactor("patch", "pods", func(action clienttesting.Action) (bool, runtime.Object, error) {
```

Comment:
it's surprising these don't need to handle the patch/delete requests in any way

Reply:
This unit test focuses asserts on nominating the node for preemption. It does not assert the modifications done to the pod so it ignores the DELETE (and now also PATCH) action. 

However, a simple (yet more powerful alternative) is to index the potential victim pods in the fake client so that the test does not fail trying to patch non-existing pods (proposed in the new commit).

Reply:
@alculquicondor @soltysh  PTAL